deploy/007-deploy-vaults.ts

```diff
@@ -0,0 +1,67 @@
+import { ethers } from 'hardhat';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import deployedContracts from '@venusprotocol/governance-contracts/deployments/deployments.json'
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const xvsDeployment = await deploy('XVS', {
```

Comment:
I would deploy XVS in a previous script. This way we can define the address of the XVS token to be considered on each network. For example, in Ethereum the XVS will be the bridged XVS deployed [here](https://github.com/VenusProtocol/isolated-pools/pull/291)